packages/core/README.md

```diff
@@ -375,3 +402,28 @@ map.$store.watch(
 This is, for example, useful to listen to search results, draw features, or marker coordinates. The plugins document how exactly to use their respective fields.
 
 To add content to the `MoveHandle`, the mutation `setMoveHandle` can be used. The values needed are described in `@polar/lib-custom-types:MoveHandleProperties`.
+
+### Mutations
+
+#### setTheme
+
+This can be used to change the theme.
+
+```js
+setTheme(state, theme) {
+        state.theme = theme
+      },
+```
+
+| fieldname | type | description |
+| - | - | - |
+| theme | enum['light', 'dark'] | By default the theme is set to 'light'. Setting theme to 'dark' will render the default 'dark' theme. |
+
+### Actions
+
+```js
+changeTheme({ commit }, theme) {
+        commit('setTheme', theme)
+        recolor(theme)
+      },
+```
```

Comment:
```suggestion
```
`snowbox` uses the `mutation` and the pattern you implemented wants the `mutation` to be the primary function to be called by users. So I don't think we should document this.